### ISSUE TITLE
method for using Wazuh info as IPS?

### ISSUE THREAD

**Issue by airdogvan** [wazuh/wazuh-docker#379]
Wazuh sends lots of warnings and I'm wondering if there's a way to use them to use some of those to update firewall rules.

For instance 

**rule.mitre.technique | Brute Force**

was on some of the alerts that are emitted after user tried to login repeatedly with non existent credentials.

It would be nice if there was a way of banning this IP with the firewall as a response to the warning.  The Wazuh message includes all the relevant info, it's all there, just need to use it.

Any suggestions welcomed.

**Comment by Zenidd** [comment]
Hello @airdogvan 

I think the Wazuh Active response feature fits your use case. It executes a script in response to the triggering of specific alerts based on the alert level or rule group. Any number of scripts can be initiated in response to a trigger, however, these responses should be considered carefully. Poor implementation of rules and responses may increase the vulnerability of the system.

Information about the use and configuration of active-response:
https://documentation.wazuh.com/3.13/user-manual/capabilities/active-response/
You can develop your own scripts:
https://documentation.wazuh.com/3.13/user-manual/capabilities/active-response/remediation-faq.html#can-i-use-a-custom-script-for-active-responses
Check our blog on blocking attacks with active response
https://wazuh.com/blog/blocking-attacks-active-response/


Do not hesitate to ask further questions. Greetings,

JP

**Comment by manuasir** [comment]
Hello @airdogvan 
Was that useful for you? Please let me know if I can already close this ticket.

Regards

**Comment by airdogvan** [comment]
Sorry @manuasir I totally forgot to answer this one.  Yes it was useful, as a matter of fact after posting this I went back to the docs and reviewed the active response section.  Implemented a rule similar to fail2ban for brute force attacks just to see how it would work and it seems to work well.

And then I read your post and wanted to answer but something else took me away and the forgot.

Again sorry about not closing it earlier, this one slipped through the cracks.

Thanks so much once again for the help.
